test/Test/Arb.juvix

```diff
@@ -20,23 +19,20 @@ arbitrarySizedList
                   rSplit : StdGen × StdGen := stdSplit r;
                   r1 : StdGen := fst rSplit;
                   r2 : StdGen := snd rSplit;
-                in runGen g r1 :: randList r2 n;
+                in Gen.run g r1 :: randList r2 n;
           in randList rand size});
 
 --- Generate an arbitrary rectangular matrix
-arbitraryMatrix
-  : {A : Type} -> Arbitrary A -> Arbitrary (List (List A))
-  | {A} arbA :=
-    arbitrary
-      (gen
+arbitraryMatrix {A} (arbA : Arbitrary A) : Arbitrary (List (List A)) :=
```

Comment:
Is this required, because we have an `Arbitrary (List A)` instance now? https://github.com/anoma/juvix-quickcheck/blob/0a4cae9daba9e495f1bec00ddec8649c37f25290/Test/QuickCheck/Arbitrary.juvix#L36

Reply:
This is needed to generate matrices (which have the same length of each row).